backend/src/EventList.WebApi/Common/Behaviors/UnhandledExceptionBehaviour.cs

```diff
@@ -17,6 +18,10 @@ public async Task<TResponse> Handle(TRequest request, RequestHandlerDelegate<TRe
             {
                 return await next();
             }
+            catch (ValidationException ex)
```

Comment:
ValidationException inherits from Exception anyway, is this extra catch required? We would need to duplicate try catch we already have in the middleware.

Reply:
Remove not neccessary code